app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user, optional: true
+  belongs_to :post, optional: true
```

Comment:
- [ ] Since you have `author_id` as a reference column for User Model in your Like Models, then in setting the association you use the below code to correctly point to the User class 👇🏾 

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```
The `class_name` option point to the Model that should be used for the association since your column name is `author_id`

Reply:
Fixed.